app/controllers/dashboard/protocol_merges_controller.rb

```diff
@@ -55,13 +55,16 @@ def perform_protocol_merge
         @errors[:merged_protocol_id] = t(:dashboard)[:protocol_merge][:errors][:one_calendar]
       elsif Setting.get_value("fulfillment_contingent_on_catalog_manager") && @merged_protocol.fulfillment_protocols.any?
         @errors[:merged_protocol_id] = t(:dashboard)[:protocol_merge][:errors][:cannot_merge]
+      elsif requests_not_merge_eligible?(@master_protocol, @merged_protocol)
+        @errors[:master_protocol_id] = t(:dashboard)[:protocol_merge][:errors][:split_notify_error]
       elsif @errors.empty? && !confirmed
         @no_errors = true
         return
       else
         ActiveRecord::Base.transaction do
+          merged_ssr_ids = @merged_protocol.sub_service_requests.map(&:id)
           merge_srs = Dashboard::MergeSrs.new()
-          fix_ssr_ids = Dashboard::FixSsrIds.new(@master_protocol)
+          fix_ssr_ids = Dashboard::FixSsrIds.new(@master_protocol, merged_ssr_ids)
```

Comment:
You could pass the `merged_ssrs` directly instead of the IDs to avoid having to re-find them in the `FixSsrIds` file

Reply:
I think I'm going to have to leave this one as is. I can explain more offline if you'd like, but it has to do with the ssr's protocol id changing with the merge.